src/Components/Location/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from "react";
+import { Typography } from "@mui/material";
```

Comment:
Check how you have imported Typography in other files why is this one imported like this

Reply:
Done